r120/outref/ydb_dbglvl.txt

```diff
@@ -0,0 +1,10 @@
+# Test for ydb_dbglvl env var
+# Test that gtmdbglvl=2 (print usage statistics at process end) controls ydbDebugLevel if ydb_dbglvl is undefined
```

Comment:
Why use ydbDebugLevel here? The name of the internal variable is still gtmDebutLevel - likely for a long time to come.

Reply:
To address this, I have made the code changes to say it is ydbDebugLevel in the simpleAPI branch (#59 which also houses the code changes for #115).

Reply:
I'm still missing something as my confusion runs rampant. The gtmDebugLevel variable is a C global variable not accessible from M code so unless you changed it yesterday and I don't see it yet, then it is still gtmDebugLevel even in the simpleAPI version so has not been changed, nor is there any reason to change the global variable name.

Reply:
It was a typo in a comment line in the test script. And so it did not matter whether it was ydbDebugLevel or gtmDebugLevel. Now that I have made the ydbDebugLevel change, what was a typo is no longer one. Hope this clears your confusion.